visidata/data.py

```diff
@@ -160,7 +162,7 @@ def openPath(vd, p, filetype=None):
     return openfunc(p)
 
 
-@VisiData.global_api
+@VisiData.api
```

Comment:
keep as `global_api` for now, we'll make that be the deprecation mechanism and clean it up later.  (this way other plugins/etc using it as a global should still work).